otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
+      description: "Wallet DIM stub application",
+      allow_forking: true,
```

Comment:
should not be necessary, as the value defaults to `true`

Reply:
Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/commit/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27